coinbase_check_v2.sh

```
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Direct Coinbase API check via curl with full paths
+
+set -e
+
+# Load environment
+export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)